projects/admin/src/app/service/app-init.service.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2020 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Injectable } from '@angular/core';
+import { LocalStorageService, TranslateService } from '@rero/ng-core';
+import { User } from '../class/user';
+import { AppConfigService } from './app-config.service';
+import { OrganisationService } from './organisation.service';
+import { UserService } from './user.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AppInitService {
+
+  /**
+   * Constructor
+   * @param _userService - UserService
+   * @param _appConfigService - AppConfigService
+   * @param _translateService - TranslateService
+   * @param _organisationService - OrganisationService
+   * @param _localeStorageService - LocalStorageService
+   */
+  constructor(
+    private _userService: UserService,
+    private _appConfigService: AppConfigService,
+    private _translateService: TranslateService,
+    private _organisationService: OrganisationService,
+    private _localeStorageService: LocalStorageService
+  ) { }
+
+  /** Application load */
+  load() {
+    return new Promise((resolve) => {
```

Comment:
Can you use an Observable here to be consistent with the rest of the application? Else ignore my comment.

Reply:
Remove Promise (not necessary)